vm/systemSmartContracts/eei.go

```diff
@@ -389,10 +389,11 @@ func (host *vmContext) ExecuteOnDestContext(destination []byte, sender []byte, v
 		return nil, err
 	}
 
+	vmOutput := &vmcommon.VMOutput{}
```

Comment:
Why it was moved from line 415 up here? It is still not used above that line.

Reply:
I thought to use in defer.